portfolio/src/main/webapp/style.css

```diff
@@ -18,4 +18,21 @@
     height: auto;
 }
 
+.image-gallery {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.image-gallery img {
+  margin: 10px;
+  max-height: 200px;
+  max-width: 200px;
+}
+
+.image-gallery img:hover {
+  border: thick solid black;
+}
+
+
```

Comment:
nit (meaning something minor, not mandatory to fix):
with this change there will be 3 blank lines at the end of the file - consider keeping it constant. we usually keep 1 blank line at the end.